docs/features/boolean-product-attribute.md

```diff
@@ -0,0 +1,17 @@
+# BooleanProductAttributeField
+
+This is currently unsupported — while the attributes do show up on the product's page, they serve no purpose for the time being.
```

Comment:
You can still use it to indicate something about the line item so it has a purpose. As far as I see right now if you add one to the product it doesn't actually show up in the order, but that's a bug and shouldn't be considered for this documentation. I will open a separate issue for it after I'm done with the review. So we can write something like:

> Adds a checkbox so the buyer can communicate some kind of yes/no choice about their purchase. Currently this attribute is not supported for price variant creation.

Same applies to NumericProductAttributeField as well